dotCMS/src/main/java/com/dotcms/rest/ContentResource.java

```diff
@@ -463,6 +464,47 @@ public Response getContent(@Context HttpServletRequest request,
         return responseResource.response(result, null, status);
     }
 
+    /**
+     * This methods receives a Lucene query.
+     * It processes the query looking for special scenarios like structure fields (i.e: stInode, stName) and replace them with valid Content fields
+     * @param luceneQuery
+     * @return luceneQuery
+     */
+    private String processQuery(String luceneQuery) throws DotDataException, DotSecurityException {
+        if (luceneQuery == null) {
+            return null;
+        }
+
+        //Lucene parameters are separated by blankSpace
+        String [] params = luceneQuery.split(" ");
+        for (String paramStr : params) {
+
+            if (paramStr.contains(Contentlet.STRUCTURE_INODE_KEY + ":")) {
+                //Parameter is in the FORMAT  stInode:inode
+                String[] param = paramStr.split(":"); //Param and value pair separated by :
+                if (param.length == 2) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 2 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS109)